radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/addressbook/AddressBook.java

```diff
@@ -229,10 +225,44 @@ private boolean sameNetworkHrp(RadixNodeUri uri) {
   }
 
   public void addOrUpdatePeerWithSuccessfulConnection(RadixNodeUri radixNodeUri) {
+    this.cleanupPeerEntriesFromRemoteHostPort(radixNodeUri.getHost(), radixNodeUri.getPort());
```

Comment:
I think we might only want to do this if `peerChannel.isInitiator == true`, otherwise the port is just a randomly assigned one

Reply:
Fixed in https://github.com/radixdlt/radixdlt/pull/652